host/libhackrf/src/hackrf.h

```diff
@@ -27,31 +27,31 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
 #include <stdint.h>
 
 #ifdef _WIN32
-   #define ADD_EXPORTS
-   
-  /* You should define ADD_EXPORTS *only* when building the DLL. */
-  #ifdef ADD_EXPORTS
-    #define ADDAPI __declspec(dllexport)
-  #else
-    #define ADDAPI __declspec(dllimport)
-  #endif
-
-  /* Define calling convention in one place, for convenience. */
-  #define ADDCALL __cdecl
+#define ADD_EXPORTS
+
+/* You should define ADD_EXPORTS *only* when building the DLL. */
+#ifdef ADD_EXPORTS
+#define ADDAPI __declspec(dllexport)
+#else
+#define ADDAPI __declspec(dllimport)
+#endif
```

Comment:
There is an option to indent statements inside an `#ifdef` block if we want that - it wasn't something we were consistent about before so I left it off.

Reply:
Even though the nesting of defines in hackrf isn't complex, I find the indentation made it very clear. (Was a bit surprised that it was removed.) Some points about indenting defines: https://stackoverflow.com/questions/789073/indenting-defines

Reply:
I'm fine with adding it to the places we were missing it. Have gone ahead and done that - I had to manually help it indent some multiline comments within `#ifdef` blocks though.